loader/src/main/java/net/neoforged/fml/loading/LoadingModList.java

```diff
@@ -92,9 +92,9 @@ public void addEnumExtenders() {
         modFiles.stream()
                 .map(ModFileInfo::getMods)
                 .flatMap(List::stream)
-                .forEach(mod -> mod.getConfig().<String>getConfigElement("enumExtender").ifPresent(file -> {
+                .forEach(mod -> mod.getConfig().<String>getConfigElement("enumExtensions").ifPresent(file -> {
                     Path path = mod.getOwningFile().getFile().findResource(file);
-                    if (Files.notExists(path)) {
+                    if (!Files.isRegularFile(path)) {
                         ModLoader.addLoadingIssue(ModLoadingIssue.error("fml.modloading.enumextender.file_not_found", path).withAffectedMod(mod));
```

Comment:
I feel like it should have a better error message if for some reason it's a directory.

Reply:
Very well then